agreements/membership/membership-renewal.yaml

```diff
@@ -0,0 +1,29 @@
+title: "Commonhaus Foundation Membership Renewal"
+version: "initial draft"
+preamble: |
+  I would like to renew my membership to the Commonhaus Foundation (CF) for another year.
```

Comment:
Not sure how I feel about people having to fill out useless paperwork once a year.

Reply:
Indeed. We'd like to know when members become inactive. Perhaps instead of asking for renewals we ping them for a given number of times, using automation of course.

Reply:
I could use this only for re-activation. In my mind, this was a (fairly painless) way of making sure people are aware if there were changes to policies. From a "how is this document actually presented" (on the website). I can avoid showing it if all language is the same (which is the purpose of the version). If terms have changed (additional agreement, e.g., because we rely on an additional service that has its own ToS), then we should show that, too.

Remember, this is text for a membership management tool. There are lots of ways to make sure we show this in a useful way. That said, I'll take out the "for another year" sentence.

Reply:
A side perspective, Why the hesitancy in requiring a few minutes once a year to overview changes and re-affirm a CF commitment? Initial commitment should be a mindful decision, as should remaining in CF. We're not a gym membership to subscribe and forget about until you realize you want a refund. 💸 

That said, if there is a move to make the renewal process the equivalent of 30 pages, then we can start throwing 😭 emoji.